deptofdefense/move.mil
# [WIP] Weight estimator

## Checklist

I have…


## Summary of Changes

This is a WIP for the weight estimator page. First time creating a model / controller / seeding data / writing spec tests etc so looking for some high level feedback. There is currently also no design so the UI is very rough and will change.
Weight data is extracted + parsed from the DPS weight calculator.

For issue #70

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
2. git checkout weight-estimator
3. set up development dependencies according to CONTRIBUTING.md,
4. run bin/rails db:drop db:create db:migrate db:seed,
5. run bin/rails server, and
6. load up http://localhost:3000/resources/weight-estimator in your Web browser of choice,
7. enter some quantities and click calculate

## Screenshots
When first landing on the page:
<img width="1059" alt="screen shot 2017-10-11 at 5 23 29 pm" src="https://user-images.githubusercontent.com/29409348/31468050-246b1ac8-aeaa-11e7-86f7-587baa828ed0.png">

Expanding the accordion and entering quantities:
<img width="1063" alt="screen shot 2017-10-11 at 5 19 53 pm" src="https://user-images.githubusercontent.com/29409348/31468060-2f19a1ba-aeaa-11e7-89d6-c674aff55e3e.png">

Click calculate button:
<img width="1135" alt="screen shot 2017-10-11 at 5 20 17 pm" src="https://user-images.githubusercontent.com/29409348/31468066-36ab032e-aeaa-11e7-9094-141cd854f6a5.png">
